client/sites/staging-site/components/staging-site-card/index.js

```diff
@@ -61,6 +61,7 @@ export const StagingSiteCard = ( {
 	const [ syncError, setSyncError ] = useState( null );
 	// eslint-disable-next-line no-unused-vars
 	const [ _, setIsErrorValidQuota ] = useState( false );
+	// eslint-disable-next-line no-unused-vars
```

Comment:
`progress` isn't being rendered in the UI at the moment. For now I decided to keep the state in, but I am open to removing it completely in this PR.

Reply:
Two things:

* If there's unused state, let's remove it
* If we decide to keep the setters, this is valid syntax if we want to avoid the ESLint ignore:

```
const [ , setProgress ] = useState( 0.1 );
```

Reply:
Thanks, Marin.

I have decided to remove the state completely (https://github.com/Automattic/wp-calypso/pull/101230/commits/39c12ec8e2d541dac675587075e2a56ba87debed).